README.md

```diff
@@ -12,6 +12,15 @@ See also:
 ## [License](https://github.com/Microsoft/MSIX-PackageSupportFramework/blob/master/LICENSE)
 Code licensed under the [MIT License](https://github.com/Microsoft/MSIX-PackageSupportFramework/blob/master/LICENSE).
 
+## Branch structure
+Package Support Framework adopts a development and master branching style.
+
+### Master
+This branch is what our NuGet repository uses.  The code in this branch is the most stable.  Please do not form off this branch for development.
```

Comment:
Branch that represents the source that exists in the latest NuGet package.

nit:form->fork